Partition (WBP)/Functions/VI0034C.VTI0034.fun.yml

```diff
@@ -36,11 +35,16 @@
     Clr_List Named(#wLRet)
     #Com_Owner.mthAdd_Msg( OK *Function ('Start - ' + *Function_Desc) )
     
+    #Com_Owner.mthGetExecutionType()
+    
     Get_Entry Number(1) From_List(#wLKData)
     
     *
     * Add your test case here
     * It should be a list of method calls so its easy to read whats being tested.
+    If (#LF_Execution_Type *EQ Q)
+    #Com_Owner.mthAdd_Msg( OK ('Skipping SuperServer IO test due to defect described in #34') ('') )
```

Comment:
Use the ticket number I gave you #34 does not describe this defect.

Reply:
As ticket points mentioned that IBM-i are not to be tested Thought to mention #34.

Now its corrected to CCS ticket number

Reply:
updated the correct ticket number.